utils/apply_errata

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
All shebangs are now changed to `python3` as we don't support `python2` anymore.